app/roles/management/tasks/netboot/kvm.yml

```diff
@@ -17,5 +19,5 @@
     virsh vol-create-as --pool default --name {{ inventory_hostname }}.qcow2
     --capacity {{ kvm_vol_cap.stdout.split('.')[0] }}G --format qcow2 &&
     virsh start {{ inventory_hostname }}
-  delegate_to: "{{ management_hostname }}"
-  become: yes
+  delegate_to: "{{ management_hostname }}" # noqa no-changed-when
```

Comment:
maybe best to have a `changed_when: true` on this task?

Reply:
If you prefer that, I can do that instead